ikisocket/go.mod

```diff
@@ -1,11 +1,11 @@
-module github.com/antoniodipinto/ikisocket
+module github.com/gofiber/contrib/ikisocket
 
-go 1.18
+go 1.20
 
 require (
 	github.com/fasthttp/websocket v1.5.4
 	github.com/gofiber/contrib/websocket v1.2.0
-	github.com/gofiber/fiber/v2 v2.50.0
+	github.com/gofiber/fiber/v2 v2.48.0
 	github.com/google/uuid v1.3.1
```

Comment:
This should had gone up to v2.52.0 not down to 2.48 😁

Reply:
Wrong go get, solved now